examples/config-install/values.yaml

```diff
@@ -0,0 +1,7 @@
+operator.image: "zufardhiyaulhaq/ngrok-operator"
```

Comment:
I don't think we need this, the example values is already in `charts/ngrok-operator/values.yaml`

Reply:
ok sorry  i dont not viewed this file i delete this file new